<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Elija una opcion:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="opcion1">
         <property name="text">
          <string>Opcion 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opcion2">
         <property name="text">
          <string>Opcion 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opcion3">
         <property name="text">
          <string>Opcion 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Elija otra opcion:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="opciona">
         <property name="text">
          <string>Opcion A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opcionb">
         <property name="text">
          <string>Opcion B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="opcionc">
         <property name="text">
          <string>Opcion C</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mensaje">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
